src/collections/lookup-map.ts

```diff
@@ -12,7 +12,7 @@ export class LookupMap<DataType> {
 
   get(key: Bytes, options?: GetOptions<DataType>): DataType | null {
     const storageKey = this.keyPrefix + key;
-    const value = JSON.parse(near.storageRead(storageKey));
+    const value = deserialize(near.storageRead(storageKey));
```

Comment:
It's great to make serialize / desalinize general / independent of the collections! Since we have `GetOptions`, it's possibly a good idea to make it as part of `GetOptions`, and have current `utils.serialize`/`utils.deserialize` as the default option.

Reply:
@ailisp I have covered this in the latest commits. Let's discuss the approach I took there. I also added a `serializer` and `deserializer` option to the `NearBindgen` decorator

Reply:
That looks great!